docs/api/vast-tracker.md

```diff
@@ -68,8 +68,9 @@ Here is the list of events emitted by the class:
 
 ## Public Methods 💚 <a name="methods"></a>
 
-### errorWithCode(errorCode)
+### errorWithCode(errorCode, isCustomCode)
 Sends a request to the URI provided by the VAST `<Error>` element. If an `[ERRORCODE]` macro is included, it will be substituted with `errorCode`.
+Use the `isCustomCode` Boolean to avoid checking `[ERRORCODE]` during the macro remplacement and allow any value.
```

Comment:
Please add the parameter to the list of the parameters just below

Reply:
I'd say better:
> Pass `isCustomCode` as `true` in order to use any value. If `false` or no value is passed, the macro will be replaced using `errorCode` only if the code is a number between `100` and `999` (see https://gist.github.com/rhumlover/5747417). Otherwise `900` will be used.